### ISSUE TITLE
Fatal Exception: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/chromium/net/ExperimentalCronetEngine$Builder;

### ISSUE THREAD

**Issue by kartikzun** [TuyaInc/tuyasmart_home_android_sdk#185]
Hi, 

We use the tuya smart android SDK in our app and have started to receive the following exception in a long running android app. This has started happening since today with no new updates from our end to the app and is crashing our android app. Could you please check - 

```
Fatal Exception: java.lang.NoClassDefFoundError
Failed resolution of: Lorg/chromium/net/ExperimentalCronetEngine$Builder


Fatal Exception: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/chromium/net/ExperimentalCronetEngine$Builder;
       at com.tuya.smart.android.network.quic.TuyaSmartQuicManager.createCronetEngine(:2)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher$1$1.onSuccess(:48)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher.requestCerts(:103)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher.access$300()
       at com.tuya.smart.android.network.http.pin.CertPinRefresher$1.run(:7)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
       at java.lang.Thread.run(Thread.java:764)

Caused by java.lang.ClassNotFoundException
org.chromium.net.ExperimentalCronetEngine$Builder
com.tuya.smart.android.network.quic.TuyaSmartQuicManager.createCronetEngine

Caused by java.lang.ClassNotFoundException: org.chromium.net.ExperimentalCronetEngine$Builder
       at com.tuya.smart.android.network.quic.TuyaSmartQuicManager.createCronetEngine(:2)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher$1$1.onSuccess(:48)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher.requestCerts(:103)
       at com.tuya.smart.android.network.http.pin.CertPinRefresher.access$300()
       at com.tuya.smart.android.network.http.pin.CertPinRefresher$1.run(:7)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:919)
```

**Comment by Lelight-Iot** [comment]
update to 'com.tuya.smart:tuyasmart:3.17.10'

**Comment by kartikzun** [comment]
ok, but it seems this was a temporary issue and is resolved now with no change to 3.17.0 as well

Thanks
